dev-python/m2crypto/m2crypto-0.31.0.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{4..7})
+PYTHON_REQ_USE="threads(+)"
+
+inherit distutils-r1
+
+MY_PN="M2Crypto"
+
+DESCRIPTION="M2Crypto: A Python crypto and SSL toolkit"
```

Comment:
`${PN}` shall not be repeated in description.

Reply:
OK, that was just copied from the old version, but fixing.